### DISCUSSION TITLE
How to make a control description list

### DISCUSSION THREAD

**Discussion by TomBarkas** [OpenOrienteering/mapper#1628] [Questions & Answers] [answered]
### Steps to reproduce
1.
2.
3.

### Actual behaviour
Tell us what happens

### Expected behaviour
Tell us what should happen instead

### Configuration
Mapper Version:
Operating System:

**Comment by ghost** [comment, marked as answer]
1. Create new map using "`Course_Design_<scale>`" symbol set (where `<scale>` is your original map scale, e.g. 4000, 5000, 10000 or 15000):
- ![pic.1](https://i.imgur.com/CQl8BYN.png)
2. Follow instruction described in «Mapper Manual»:
  - https://www.openorienteering.org/mapper-manual/pages/course_design.html

**Comment by TomBarkas** [comment]
There are no instructions on creating a control description list, only what you can put in it, which is already obvious

Tom Barkas

**Comment by TomBarkas** [comment]
What I mean is how do you create the basic grid that you then insert the details in to

Tom Barkas

**Comment by ghost** [comment]
> What I mean is how do you create the basic grid that you then insert the details in to

When you create new map using "Course_Design_<scale>"  it already include predefined grid according next notes from «Mapper Manual»:

> The control description symbols are meant to be used with a grid of horizontal and vertical lines spaced 3.0 millimeters. By pressing the <kbd>Shift</kbd> key while creating a new element, you may easily align all the parts of the description.

The only thing you should do (if grid not immediately activated) — just enable grid manually:

- using <kbd>G</kbd> key;
- using <kbd>![Show grid](https://raw.githubusercontent.com/OpenOrienteering/mapper/master/images/grid.png)</kbd> button on *"View"* toolbar;
- using menu *"View > Show grid"*.

Then create description table using symbols: `99.0`.. `99.5`, `13.1`.. `14.3`.
